packages/shop/src/components/banner/Banner.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
```

Comment:
<img width="439" alt="image" src="https://user-images.githubusercontent.com/13868235/200092595-8f8ae998-f1b8-480e-8dab-5a5fcfdf8f6b.png">
The size of inventory and banner image are different. Is it just because of the wrong campaign setting?

Reply:
It's because the creative image that the mocked response uses has different size from the inventory size. Was intentional to test the image display strategy.